src/cli/download/data-set/DataSet.definition.ts

```diff
@@ -45,7 +45,8 @@ export const DownloadDataSetDefinition: ICommandDefinition = {
             type: "string"
         },
         {
-            name: "rdw",
+            name: "record",
+            aliases: ["rdw"],
```

Comment:
Should this also contain the 'r' alias for parity with the z/OSMF command options?